### PR TITLE
feat: remoteadd in client data

### PR DESCRIPTION
**Pull request by amsjavan**
Add remoteAdd field to ClientData, AuthorizedClientData and GuestClientData classes

### CODE REVIEW

File: src/main/scala/im/actor/api/ApiServiceTrees.scala [hunk 1]
```diff
@@ -44,11 +43,12 @@ private[api] trait ApiServiceTrees extends TreeHelpers with StringHelperTrees {
           PARAM("sessionId", LongClass),
           PARAM("userId", IntClass),
           PARAM("authSid", IntClass),
-          PARAM("appId", IntClass)
+          PARAM("appId", IntClass),
+          PARAM("remoteAdd", optionType(valueCache("String"))) := REF("None")
```
Comment (onsails) [inline src/main/scala/im/actor/api/ApiServiceTrees.scala]: remoteAddr

**Comment by onsails** [comment]
Please resolve conflicts. I've pushed the latest version right now.

**Comment by amsjavan** [comment]
Resolved.

**Comment by onsails** [comment]
Published new version.

**Pull request merged**